### DISCUSSION TITLE
A place to add a comment on flows

### DISCUSSION THREAD

**Discussion by aprilahijriyan** [postmanlabs/postman-flows#340] [Ideas]
Hello!

I want to put forward the idea of adding "comments" to the flow, I'm currently using the feature of adding text via buttons. But I don't think it's a good idea, because the description of one flow might be long.

<img width="920" alt="image" src="https://user-images.githubusercontent.com/37798612/175835381-d8f87c81-fa7b-4b18-9459-533d92dd3509.png">

The idea I propose here, maybe it's good to add a comment when we click on the flow details and fill in the "comments" field.

Thank you!

**Comment by shrey-bhagat** [comment]
@aprilahijriyan thanks for the suggestion. We are already working on introducing commenting in flows.
